### PR TITLE
Updates in TPM 3.1

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -242,8 +242,9 @@ To accomplish the task, the filter string is first constructed and resolved loca
 ## PARAMETERS
 
 ### -Credential
-This parameter is not used with Skype for Business Online and will be deprecated in the near future.
 
+  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longet relevant to Microsoft Teams*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:246]:
```suggestion
  *This parameter has been deprecated from Teams Powershell Module 3.0 and later as it is no longer relevant to Microsoft Teams*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -258,8 +259,9 @@ Accept wildcard characters: False
 ```
 
 ### -DomainController
-This parameter is not used with Skype for Business Online and will be deprecated in the near future.
 
+  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longet relevant to Microsoft Teams*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:263]:
```suggestion
  *This parameter has been deprecated from Teams Powershell Module 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 3]
```diff
@@ -318,6 +320,9 @@ Accept wildcard characters: False
 ```
 
 ### -LdapFilter
+ 
+  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longet relevant to Microsoft Teams*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:324]:
```suggestion
  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 4]
```diff
@@ -338,6 +343,9 @@ Accept wildcard characters: False
 ```
 
 ### -OnModernServer
+ 
+ *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:347]:
```suggestion
 *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 5]
```diff
@@ -355,7 +363,8 @@ Accept wildcard characters: False
 ```
 
 ### -OnOfficeCommunicationServer
-This parameter is not used with Skype for Business Online and will be deprecated in the near future.
+ 
+ *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longet relevant to Microsoft Teams*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:367]:
```suggestion
 *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 6]
```diff
@@ -371,7 +380,9 @@ Accept wildcard characters: False
 ```
 
 ### -OU
-This parameter is not used with Skype for Business Online and will be deprecated in the near future.
+ 
+ *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longet relevant to Microsoft Teams*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:384]:
```suggestion
 *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 7]
```diff
@@ -409,6 +420,9 @@ Accept wildcard characters: False
 ```
 
 ### -SkipUserPolicies
+ 
+ *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:424]:
```suggestion
 *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 8]
```diff
@@ -425,6 +439,9 @@ Accept wildcard characters: False
 ```
 
 ### -UnassignedUser
+ 
+ *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:443]:
```suggestion
 *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 9]
```diff
@@ -442,6 +459,9 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
+ 
+ *These parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:463]:
```suggestion
 *These parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 10]
```diff
@@ -37,6 +37,19 @@ Beginning Teams PowerShell Module 2.6.2 onwards, the below updates are applicabl
   - ResultSize
 - LicenceState 
 
+*Deprecated Input Parameters*
+
+- Below input parameters have been deprecated due to low/zero usage:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:42]:
```suggestion
- The following input parameters have been deprecated due to low or zero usage:
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 11]
```diff
@@ -134,6 +147,8 @@ Accept wildcard characters: False
 ```
 
 ### -Force
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:150]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 12]
```diff
@@ -152,6 +167,8 @@ Accept wildcard characters: False
 ```
 
 ### -GetFromAAD
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:170]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 13]
```diff
@@ -168,6 +185,8 @@ Accept wildcard characters: False
 ```
 
 ### -GetPendingUsers
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:188]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 14]
```diff
@@ -256,6 +275,8 @@ Accept wildcard characters: False
 ```
 
 ### -PSTNConnectivity
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:278]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 15]
```diff
@@ -275,6 +296,8 @@ Accept wildcard characters: False
 ```
 
 ### -SearchQuery
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:299]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 16]
```diff
@@ -311,6 +334,8 @@ Accept wildcard characters: False
 ```
 
 ### -Tenant
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:337]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 17]
```diff
@@ -327,6 +352,8 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
+*These parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:355]:
```suggestion
*These parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.

**Comment by isrumnon** [comment]
@JohanFreelancer9  committed the suggested changes. Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by dariomws — APPROVED** [review]
@isrumnon, could you kindly double-check if this note should be added to -CommonParameters? Thanks!

**Comment by isrumnon** [comment]
@dariomws done, made appropriate changes. Thanks!

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 18]
```diff
@@ -242,8 +242,9 @@ To accomplish the task, the filter string is first constructed and resolved loca
 ## PARAMETERS
 
 ### -Credential
-This parameter is not used with Skype for Business Online and will be deprecated in the near future.
 
+  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:246]:
```suggestion
  *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 19]
```diff
@@ -258,8 +259,9 @@ Accept wildcard characters: False
 ```
 
 ### -DomainController
-This parameter is not used with Skype for Business Online and will be deprecated in the near future.
 
+  *This parameter has been deprecated from Teams Powershell Module 3.0 and above as it is no longer relevant to Microsoft Teams*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:263]:
```suggestion
  *This parameter has been deprecated from Teams PowerShell Module 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 20]
```diff
@@ -318,6 +320,9 @@ Accept wildcard characters: False
 ```
 
 ### -LdapFilter
+ 
+  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:324]:
```suggestion
  *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 21]
```diff
@@ -134,6 +147,8 @@ Accept wildcard characters: False
 ```
 
 ### -Force
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:150]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 22]
```diff
@@ -311,6 +334,8 @@ Accept wildcard characters: False
 ```
 
 ### -Tenant
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:337]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 23]
```diff
@@ -275,6 +296,8 @@ Accept wildcard characters: False
 ```
 
 ### -SearchQuery
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:299]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 24]
```diff
@@ -256,6 +275,8 @@ Accept wildcard characters: False
 ```
 
 ### -PSTNConnectivity
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:278]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 25]
```diff
@@ -168,6 +185,8 @@ Accept wildcard characters: False
 ```
 
 ### -GetPendingUsers
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:188]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 26]
```diff
@@ -152,6 +167,8 @@ Accept wildcard characters: False
 ```
 
 ### -GetFromAAD
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:170]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 27]
```diff
@@ -425,6 +439,9 @@ Accept wildcard characters: False
 ```
 
 ### -UnassignedUser
+ 
+ *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:443]:
```suggestion
 *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 28]
```diff
@@ -409,6 +420,9 @@ Accept wildcard characters: False
 ```
 
 ### -SkipUserPolicies
+ 
+ *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:424]:
```suggestion
 *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 29]
```diff
@@ -371,7 +380,9 @@ Accept wildcard characters: False
 ```
 
 ### -OU
-This parameter is not used with Skype for Business Online and will be deprecated in the near future.
+ 
+ *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:384]:
```suggestion
 *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 30]
```diff
@@ -355,7 +363,8 @@ Accept wildcard characters: False
 ```
 
 ### -OnOfficeCommunicationServer
-This parameter is not used with Skype for Business Online and will be deprecated in the near future.
+ 
+ *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:367]:
```suggestion
 *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 31]
```diff
@@ -338,6 +343,9 @@ Accept wildcard characters: False
 ```
 
 ### -OnModernServer
+ 
+ *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:347]:
```suggestion
 *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @isrumnon, some more, please.

**Comment by isrumnon** [comment]
@JohanFreelancer9 Done!

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 Done!

@isrumnon, thank you so much for committing the changes, but you seem to have missed two suggested changes. Could you please commit them as well? Thanks again.

**Comment by isrumnon** [comment]
@JohanFreelancer9 oh I must have missed those. Committed them, thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**